src/commands/stac-validate/stac.validate.ts

```diff
@@ -162,12 +162,12 @@ export const commandStacValidate = command({
           if (!checksum.startsWith('1220')) continue;
 
           let source = asset.href;
-          if (source.startsWith('./')) source = fsa.join(dirname(path), source.replace('./', ''));
+          if (source.startsWith('./')) source = new URL(dirname(path), source.replace('./', ''));
```

Comment:
`new URL` would work with relative paths

Reply:
Not sure what you're saying. Do you mean we should just `if (source.startsWith('./')) source = new URL(dirname(path), source)`? Both `new URL('foo', './bar')` and `new URL('foo', 'bar')` complain that the second argument is not a valid URL, so I don't understand how this code ever worked, unless we were passing crazy URLs like "./http://…".

Reply:
it would be much safer to do `new URL(path, source)` which will handle all the checking for '../../foo.json` or `/foo.json`